src/components/editor/PropertyResourceTemplate.jsx

```diff
@@ -38,13 +38,15 @@ class PropertyResourceTemplate extends Component {
       <div>
       {
         this.props.resourceTemplate.propertyTemplates.map((property) => {
+          let keyId = shortid.generate()
           let newReduxPath = Object.assign([], this.props.reduxPath)
           newReduxPath.push(property.propertyURI)
+          newReduxPath.push(keyId)
```

Comment:
Is this covered by tests?

Reply:
Yes, `PropertyTemplateOutline` now has a test for Redux Path.